nature.md

```diff
@@ -1087,7 +1091,7 @@ Third party content (e.g. advertising, customer services widgets) may inject ele
 <em>Revised Section 508 - does not apply</em>
 </td>
 <td>Partially Supports</td>
-<td>Additional content that becomes visible on hover or focus is rare in this product.
+<td>Additional content that becomes visible on hover or focus (for example tooltips) is extremely rare in this product. In all cases where content is intitally hidden and displayed on interaction, we prefer to use persistent displays that are activated by explicit, unambiguous interactions such as clicks or keypresses.
```

Comment:
```suggestion
<td>Additional content that becomes visible on hover or focus (for example tooltips) is extremely rare in this product. In all cases where content is initially hidden and displayed on interaction, we prefer to apply visibility persistently as a result of unambiguous interactions; such as clicks or keypresses.
```

"persistent displays" felt possibly ambiguous to me, whereas the concept of "visibility" is what is being discussed.

Reply:
Agree, it felt wrong as I was writing it and hasn't improved after a night's sleep and most of a working day. How do you feel about this as an alternative:

> In all cases where content is intitally hidden and displayed on interaction, we prefer to **make that content persistent until dismissed and** activated by explicit, unambiguous interactions; such as clicks or keypresses.

Reply:
Reworded after we thrashed this one out in Slack :D